.github/workflows/build.yml

```diff
@@ -102,43 +103,23 @@ jobs:
     env:
       version: ${{ needs.get-product-version.outputs.product-version }}
       arch: amd64
-      REPO_NAME: consul-lambda-registrator
```

Comment:
This is no longer effective because any value set here gets clobbered by the [`actions-docker-build`](https://github.com/hashicorp/actions-docker-build/blob/main/action.yml#L97) script. We use `PKG_NAME` instead.